src/util/Logger.ts

```diff
@@ -28,11 +28,11 @@ export class Logger {
   static addConsoleHandler = () => {
     Logger.handlers.add({
       /* eslint-disable no-console */
-      error: (label, ...args) => console.error(`[${label}] ERROR:`, ...args),
-      warn: (label, ...args) => console.warn(`[${label}] WARN:`, ...args),
-      info: (label, ...args) => console.info(`[${label}] INFO:`, ...args),
-      debug: (label, ...args) => console.debug(`[${label}] DEBUG:`, ...args),
-      debug2: (label, ...args) => console.debug(`[${label}] DEBUG2:`, ...args),
+      error: console.error.bind(console),
```

Comment:
Now the label's not being printed with the square brackets. In the `private handle` method below, wrap the `this.label` in square brackets so it's printed correctly.